org-node/src/lib.rs

```diff
@@ -410,21 +423,27 @@ async fn update_refs(mut handle: client::Handle) {
                         let stream = BufReader::new(s);
                         for urn in stream.lines().flatten() {
                             if let Ok(urn) = Urn::try_from_id(urn) {
-                                match handle.update_refs(urn).await {
+                                match handle.update_refs(urn.clone()).await {
                                     Ok(()) => {
                                         tracing::info!(target: "org-node", "Successfully updated refs");
+                                        // Notify connected websocket clients of updated refs.
+                                        if let Err(e) = ws_tx.send(webserver::WsEvent::UpdatedRef {
+                                            rev: urn.encode_id(),
```

Comment:
This isn't a rev, it's a project id. For that reason I'd encourage using the `git2::Oid` type in `WsEvent::UpdatedRef` to avoid these type errors. You can use a custom serialize function with serde: https://serde.rs/field-attrs.html#serialize_with

Reply:
commit 0452b3c fixes.